### PR TITLE
SO-424 & SO-428 - Supplier account business categories

### PR DESCRIPTION
**Pull request by adohall**

### CODE REVIEW

File: app/templates/components/seller-account/business-category-view.hbs [hunk 1]
```diff
@@ -1 +1,35 @@
...
+<h4>Business identifiers (optional)</h4>
+<div class="cool-infobox sapphire">{{fa-icon "eye" prefix="fas" size="2x"}} 
+  <div>This information will appear on your supplier profile, but only to <b>registered buyers</b>.</div>
+</div>
+<ul class="tags">
+  {{#if form.not_for_profit}}
+    <li>not for profit</li>
+  {{/if}}
+  {{#if form.start_up}}
+    <li>start up</li>
```
Comment (armansa) [inline app/templates/components/seller-account/business-category-view.hbs]: not sure, but maybe it's better to write like: "Start-up"

File: app/templates/components/seller-account/business-category-view.hbs [hunk 2]
```diff
@@ -1 +1,35 @@
...
+  {{/if}}
+  {{#if form.australian_owned}}
+    <li>australian owned</li>
```
Comment (armansa) [inline app/templates/components/seller-account/business-category-view.hbs]: Again maybe capitalise first word?

Comment (adohall) [reply app/templates/components/seller-account/business-category-view.hbs]: Agreed. Done.

File: app/templates/components/seller-account/business-category-view.hbs (hunk 1 above)
Comment (adohall) [reply app/templates/components/seller-account/business-category-view.hbs]: Agreed. Done.

**Pull request merged**